ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -438,8 +719,17 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8
+- as: vsphere-ipi-sdn-f28
```

Comment:
`sdn` is the default networkType in 4.11 (changed to `ovn` since 4.12), so I think this one is already covered by other ci jobs. from networkType coverage view.